### DISCUSSION TITLE
Dropdown animation glitch

### DISCUSSION THREAD

**Discussion by codethevoid** [heroui-inc/heroui#565] [Feedback]
When I open the dropdown, it appears right away before running the animation.

**Comment by cyanxiao** [comment]
Could you provide a sample of your work? This is one from the doc: https://codesandbox.io/s/g8hk9s

**Reply by codethevoid** [reply]
```
<Dropdown>
      <Dropdown.Trigger>
        <User
          as="button"
          name="rmthomas98"
          text="RT"
          bordered
          color="gradient"
          squared
          size="sm"
        />
      </Dropdown.Trigger>
      <Dropdown.Menu>
        <Dropdown.Item key="my-account">My account</Dropdown.Item>
        <Dropdown.Item key="settings">Settings</Dropdown.Item>
      </Dropdown.Menu>
    </Dropdown>
```

I have it exactly how it is in the documentation. But for some reason it fully shows the component right when I click the trigger and then it runs the animation after that. So you can see it for a split second before it runs the animation causing it to look glitchy.

**Comment by tianenpang** [comment]
Currently there are some issues with apparent animation in react 18, if you are using react 18, please downgrade `react` and `react-dom` to 17 and try it again.

**Comment by jrgarciadev** [comment]
Hey @rmthomas98 as @tianenpang mentioned it could be related to the `react` and `react-dom` version, currently React 18 is partially supported so it could have some issues like the dropdown animation, you could downgrade to react `v17` or disable the `Dropdown` animation using the `disableAnimation` prop

**Comment by buster95** [comment]
confirming, this issue is related to react 18, I did downgraded of react and it works fine
This is the issue I was having, looks like some kind of blink when dropdown menu appears

https://user-images.githubusercontent.com/15637669/175799131-c1a2da00-0566-4a70-bcf5-13bc1c3d58b7.mov

**Reply by codethevoid** [reply]
Yes, that's what the problem was. I downgraded to react 17 and it worked! Thanks for the help

**Comment by BuchananDevOps** [comment]
https://user-images.githubusercontent.com/110317902/183502410-ba0b54d7-b970-4001-8369-bb0cd07340f1.mp4

The issue is still present, I was able to integrate a Radix: UI Dropdown menu and apply the NextUI styles and it's been resolved

**Comment by rossanmol** [comment]
Issue still persists, is there any hack to fix this temporarily?

**Comment by jrgarciadev** [comment]
Hey please update to v2.0 https://nextui.org/blog/nextui-v2 due to v1 will not receive more updates. We moved these animations to framer-motion
